pkg/docker/docker_test.go

```diff
@@ -58,8 +58,11 @@ func TestDocker_Pull(t *testing.T) {
 	assert.Equal(t, nil, err)
 }
 
-func (t testClient) ImageBuild(ctx context.Context, context io.Reader, options types.ImageBuildOptions) (io.ReadCloser, error) {
-	return nil, nil
+func (t testClient) ImageBuild(ctx context.Context, context io.Reader, options types.ImageBuildOptions) (types.ImageBuildResponse, error) {
```

Comment:
Can we not have the ImageBuildResponse in the types.go? Can;t this be ImageBuildResponse from `docker`?

Reply:
This imagebuildresponse is from docker, we can probably get rid of our custom defined one.

Reply:
We are no longer using our own ImageBuildResponse. All commands to docker/buildah return ReadClosers which can then be read by the user. 

The ImageBuildResponse here is to implement the docker API client for mocking purposes.